code/go/0chain.net/blobbercore/challenge/challenge.go

```diff
@@ -286,7 +288,9 @@ func commitProcessed(ctx context.Context) {
 
 	rows, err := db.Model(&ChallengeEntity{}).
 		Where("status = ?", Processed).
-		Select("challenge_id", "created_at").Rows()
+		Select("challenge_id", "created_at").
+		Order("sequence").
```

Comment:
same as above
something to ensure no same challenge is taken again

Reply:
Challenge can be processed but its transaction may not be verified at that moment.
Later on it will be verified. So to update its status as `committed` it can run. It will not send challenge commit transaction to the blockchain.